action.yml

```
@@ -2,14 +2,26 @@ name: "CLI for Microsoft 365 Login"
 description: "Login to tenant using CLI for Microsoft 365"
 inputs:
   ADMIN_USERNAME:
-    description: "Username (upn) of the admin"
-    required: true
+    description: "Username (email address) of the admin"
+    required: false
   ADMIN_PASSWORD:
     description: "Password of the admin"
-    required: true
+    required: false
+  CERTIFICATE_ENCODED:
+    description: "Base64-encoded string with certificate private key"
+    required: false
+  CERTIFICATE_PASSWORD:
+    description: "Password for the certificate"
+    required: false
+  AAD_APP_ID:
+    description: "App ID of the Azure AD application to use for certificate authentication"
+    required: false
+  TENANT_ID:
+    description: "ID of the tenant from which accounts should be able to authenticate"
```

Let's align this with the option name and description from `login` command for consistency as it can accept a tenant ID or name.

```suggestion
  TENANT:
    description: "ID of the tenant from which accounts should be able to authenticate. Use common or organization if the app is multitenant. If not specified, use the tenant specified in the 'CLIMICROSOFT365_TENANT' environment variable. If the environment variable is not defined, use 'common' as the tenant identifier"
```